HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -203,6 +203,13 @@ def self.apply_header(hpxml, epw_file)
       hpxml.header.natvent_days_per_week = 3
       hpxml.header.natvent_days_per_week_isdefaulted = true
     end
+
+    if hpxml.building_occupancy.number_of_residents == 0
+      hpxml.header.vacancy_periods.add(begin_month: 1,
+                                       begin_day: 1,
+                                       end_month: 12,
+                                       end_day: 31)
+    end
```

Comment:
Not crazy about this being inside hpxml_defaults.rb. The fact that we're taking this approach is more of a modeling convenience/workaround than a default. A user seeing this in their `in.xml` would probably be confused. Perhaps move this code so that it happens right after `in.xml` is written?